app/views/content/how-to-apply-for-teacher-training/ske-explained-transcript.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Subject knowledge enhancement explained video transcript
+title: 
```

Comment:
@laurahermionetennant92 This is causing one of your build errors - you've removed the title/h1 from the SKE transcript page by mistake

Reply:
Ouch – thank you will fix